File: DentOffice.Model/Pacijent.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DentOffice.Model
{
    public class Pacijent
    {
        public int PacijentID { get; set; }
        public bool Proteza { get; set; }
        public bool Aparatic { get; set; }
        public bool Terapija { get; set; }
        public int KorisnikID { get; set; }
        public Korisnik Korisnik { get; set; }

        public override string ToString()
        {
            return Korisnik.Ime + " " + Korisnik.Prezime;
        }
    }
}

File: dentofficemobile/lib/models/pacijent.dart
// -- pacijent.dart --
import 'package:dentofficemobile/models/korisnik.dart';
import 'package:json_annotation/json_annotation.dart';

part 'pacijent.g.dart';

@JsonSerializable()
class Pacijent {
  int pacijentID;
  bool proteza;
  bool aparatic;
  bool terapija;
  int korisnikID;
  Korisnik korisnik;

  Pacijent(this.pacijentID,this.proteza,this.aparatic,this.terapija,this.korisnikID,this.korisnik,);

  factory Pacijent.fromJson(Map<String, dynamic> json) => _$PacijentFromJson(json);

  Map<String, dynamic> toJson() => _$PacijentToJson(this);
}
